central/scannerdefinitions/handler/handler.go

```diff
@@ -542,12 +593,13 @@ func (h *httpHandler) handleZipContentsFromVulnDump(ctx context.Context) error {
 	// scanner-defs.zip contains data required by Scanner V2.
 	// scanner-v4-defs-*.zip contains data required by Scanner v4.
 	// In the future, we may decide to support other files (like we have in the past), which is why we
-	// expect this ZIP of a single ZIP.
+	// loop through this ZIP of ZIPs.
 	for _, zipF := range zipR.File {
```

Comment:
The `continue`s are a bit confusing. Maybe we case use a `switch`?

Reply:
good idea